includes/class-boldgrid-inspirations-pages-and-posts.php

```diff
@@ -399,8 +399,9 @@ public function post_submitbox_misc_actions_auto_add_to_menu() {
 				 '/pages/includes/post_submitbox_misc_actions_auto_add_to_menu.php';
 
 			// Pass the selected checkboxes to the JavaScript.
+			// This was changed to an array to get rid of wp_localize_script warnings.
```

Comment:
We can probably remove this comment (and the similar one on line 530). The 3rd argument is supposed to be an array anyways.